Library/Formula/kubernetes.rb

```diff
@@ -0,0 +1,36 @@
+class Kubernetes < Formula
+  homepage "http://kubernetes.io/"
+  url "https://github.com/GoogleCloudPlatform/kubernetes/archive/v0.9.3.tar.gz"
+  sha256 "b3a6008f1db7f9153c6396cc1c3e890ca59450392556c3f823226738ab6a91fc"
+
+  bottle do
+    sha1 "049f8e24f1e33064bf763d83fbdd2374c5ac0f04" => :yosemite
+    sha1 "b06929274772957f5503e8d4d7b819a944fd066a" => :mavericks
+    sha1 "8a75ae49cc46037f748e73c78d3620287c3aa09b" => :mountain_lion
+  end
+
+  depends_on "go" => :build
+
+  def install
+    arch = MacOS.prefer_64_bit? ? "amd64" : "x86"
+
+    system "make", "all", "WHAT=cmd/*", "GOFLAGS=-v"
+
+    output_dir = "_output/local/bin/darwin/#{arch}"
+
+    bin.install "#{output_dir}/kubecfg"
+    bin.install "#{output_dir}/kubectl"
+    bin.install "#{output_dir}/kubernetes"
+  end
+
+  test do
+    output = `#{bin}/kubecfg 2>&1`.lines.to_a
```

Comment:
You can probably get rid of both the `lines` and `[0]` as `assert_match` should work without. Also, I'd consider using `system_output` instead of backticks here which will check the exit code too.

I do wonder about how worthwhile it is to just check the default outputted text, though. Is there anything we can actually do with these tools like e.g. `cmake.rb` does?